resources/SwitchMap vs MergeMap.md

```diff
@@ -0,0 +1,349 @@
+# SwitchMap vs MergeMap
```

Comment:
You end up doing a loooot more than talk about just SwitchMap vs MergeMap! I like it, it's a lot of really deep explanations about how how the 'map' operators work. We might want to expand on the introduction here and reflect the depth here in the title as well